src/UI/Http/Rest/EventSubscriber/ExceptionSubscriber.php

```diff
@@ -85,32 +87,22 @@ private function getExceptionMessage(Throwable $exception): string
 
     private function determineStatusCode(Throwable $exception): int
     {
-        // Default status code is always 500
-        $statusCode = Response::HTTP_INTERNAL_SERVER_ERROR;
-
-        switch (true) {
-            case $exception instanceof HttpExceptionInterface:
-                $statusCode = $exception->getStatusCode();
-
-                break;
-            case $exception instanceof InvalidCredentialsException:
-                $statusCode = Response::HTTP_UNAUTHORIZED;
-
-                break;
-            case $exception instanceof ForbiddenException:
-                $statusCode = Response::HTTP_FORBIDDEN;
+        $exceptionClass = \get_class($exception);
 
-                break;
-            case $exception instanceof AggregateNotFoundException || $exception instanceof NotFoundException:
-                $statusCode = Response::HTTP_NOT_FOUND;
+        foreach ($this->exceptionToStatus as $class => $status) {
+            if (is_a($exceptionClass, $class, true)) {
```

Comment:
```suggestion
            if (\is_a($exceptionClass, $class, true)) {
```

Reply:
That can be done automatically after using `make cs`.

Reply:
Maybe we should introducing some git hooks? WDYT?

Reply:
As far as CI catch this, I prefer that people in their project set up the hooks if needed.